resources/website-l10n.xsl

```diff
@@ -85,21 +85,21 @@
         <l:template name="section" text="%t"/>
       </l:context>
       <l:context name="edit-me">
-        <l:template name="edit-me-title" text="Edit this page on GitHub" />
-        <l:template name="edit-me-text" text="edit" />
+        <l:template name="edit-me-title" text="在 GitHub 上编辑本页" />
+        <l:template name="edit-me-text" text="编辑" />
       </l:context>
       <l:context name="meta">
-        <l:template name="meta-description" text="Get started with the documentation for Elasticsearch, Kibana, Logstash, Beats, X-Pack, Elastic Cloud, Elasticsearch for Apache Hadoop, and our language clients." />
+        <l:template name="meta-description" text="有关如何使用 Elasticsearch、Kibana、Logstash、Beats、X-Pack、Elastic Cloud、Elasticsearch for Apache Hadoop 及我们各种语言的客户端的文档。" />
       </l:context>
       <l:context name="annotation">
-        <l:template name="added" text="Added in " />
+        <l:template name="added" text="于 $version 添加。$custom_text" />
```

Comment:
Hi, @clintongormley the equipment stop "." in Chinese is "。"，so the new change will be like this? :
```
<l:template name="added" text="添加于" />
<l:template name="coming" text="将来自" />
<l:template name="deprecated" text="废弃于" />
 <l:template name="sentence-separator" text="。" />
```